### PR TITLE
[TASK] upgrade to V12, fix styling.

### PR DESCRIPTION
**Pull request by TimWilms**

### CODE REVIEW

File: ext_emconf.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
     'version' => '3.1.0',
     'constraints' => [
         'depends' => [
-            'typo3' => '9.5.0-11.5.99',
+            'typo3' => '9.5.0-12.4.99',
```
Comment (ruudsilvrants) [inline ext_emconf.php]: The contraint can be raised for 9.5 due the event is already not available in 9 ;)

File: Classes/EventListener/Backend/CodeSnippetPreviewRenderer.php [hunk 2]
```diff
@@ -9,39 +10,34 @@
  * of the License, or any later version.
  */
 
-use TYPO3\CMS\Backend\View\PageLayoutViewDrawItemHookInterface;
-use TYPO3\CMS\Backend\View\PageLayoutView;
+use TYPO3\CMS\Backend\View\Event\PageContentPreviewRenderingEvent;
+use TYPO3\CMS\Extbase\Utility\DebuggerUtility;
```
Comment (ruudsilvrants) [inline Classes/EventListener/Backend/CodeSnippetPreviewRenderer.php]: Remove Debugger include please

File: Classes/EventListener/Backend/CodeSnippetPreviewRenderer.php [hunk 3]
```diff
@@ -9,39 +10,34 @@
...
-            return;
-        }
-
-        $itemContent = '<strong>' . $this->getProgrammingLanguageLabel($row['programming_language']) . ':</strong><br />' .
-            '<pre><code>' . $this->prepareCodeSnippet($row['bodytext']) . '</code></pre>';
-
-        $drawItem = false;
+    public function preProcess(PageContentPreviewRenderingEvent $event): void
+    {
+        $itemContent = '<strong>' . $this->getProgrammingLanguageLabel($event->getRecord()['programming_language']) . ':</strong><br />' .
```
Comment (ruudsilvrants) [inline Classes/EventListener/Backend/CodeSnippetPreviewRenderer.php:35]: Please check the coding style guide van TYPO3, https://github.com/TYPO3/styleguide
In here you can perhaps see how the core is handling this, adjust accordenly

Comment (TimWilms) [reply Classes/EventListener/Backend/CodeSnippetPreviewRenderer.php:35]: I removed the custom old styling. if the original Ext. developer wishes to figure out how re-do the original styling, then that can be added later. TYPO3 has changed the styling in the core. This now is the new standard- if the ext. developer wishes this to be custom, he can change this himself.

**Pull request merged**